Doc/whatsnew/3.10.rst

```diff
@@ -462,6 +462,14 @@ Deprecated
   scheduled for removal in Python 3.12.
   (Contributed by Erlend E. Aasland in :issue:`42264`.)
 
+* ``sqlite3.enable_shared_cache`` is now deprecated, scheduled for removal in
+  Python 3.12.  The method is undocumented, and it's use is strongly
```

Comment:
```suggestion
  Python 3.12.  The method is undocumented, and its use is strongly
```
It is incorrect to use a contraction here.

Reply:
> Also, it seems that `sqlite3.enable_shared_cache` is not a method.

Correct, it's a built-in function. Good catch.

Reply:
See 7d52671. Is the new phrasing acceptable?